xml/decoders/kuehn/PaneShuntUncouple.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="../XSLT/decoder.xsl"?>
+<!-- version 1 - for Unified Software -->
+<pane>
```

Comment:
This should probably read:
`<pane xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">`
to pass XML validation as without, there is no declaration of _what_ this element actually is.
Hence the error from Travis:
`failed to validate "xml/decoders/kuehn/PaneShuntUncouple.xml" due to: org.jdom2.input.JDOMParseException: Error on line 4: cvc-elt.1: Cannot find the declaration of element 'pane'.`